pybamm/expression_tree/printing/print_name.py

```diff
@@ -9,6 +9,7 @@
     "current_with_time": "I",
     "timescale": r"\tau",
     "dimensional_current_with_time": r"\hat{I}",
+    "eps": r"\epsilon{c}",
```

Comment:
Why this one?

Reply:
@rtimms suggested this [here](https://pybamm.slack.com/archives/C01QNMULRB5/p1627017172018300?thread_ts=1626961961.010600&cid=C01QNMULRB5)

Reply:
Should just be \epsilon, not \epsilon{c}